src/test/scala/edu/ie3/simona/model/thermal/ThermalHouseSpec.scala

```diff
@@ -100,26 +93,26 @@ class ThermalHouseSpec extends UnitSpec with HpInputTestData {
 
     "Check build method" in {
 
-      val thermalHouse = buildThermalHouse(18, 22)
+      val thermalHouseTest = thermalHouse(18, 22)
       val thermalHouseInput = defaultThermalHouse
 
-      thermalHouse.id shouldBe thermalHouseInput.getId
-      thermalHouse.operatorInput shouldBe thermalHouseInput.getOperator
-      thermalHouse.operationTime shouldBe thermalHouseInput.getOperationTime
-      thermalHouse.bus shouldBe thermalHouseInput.getThermalBus
-      thermalHouse.ethLosses.toWattsPerKelvin shouldBe (thermalHouseInput.getEthLosses
-        .to(StandardUnits.THERMAL_TRANSMISSION)
-        .getValue
-        .doubleValue * 1000)
-      (thermalHouse.ethCapa * Temperature(
+      thermalHouseTest.id shouldBe thermalHouseInput.getId
+      thermalHouseTest.operatorInput shouldBe thermalHouseInput.getOperator
+      thermalHouseTest.operationTime shouldBe thermalHouseInput.getOperationTime
+      thermalHouseTest.bus shouldBe null
+      thermalHouseTest.ethLosses.toWattsPerKelvin shouldBe 1000.0
+      (thermalHouseTest.ethCapa * Temperature(
         1,
         Celsius,
-      )).toKilowattHours shouldBe thermalHouseInput.getEthCapa
-        .to(StandardUnits.HEAT_CAPACITY)
-        .getValue
-        .doubleValue
-      thermalHouse.lowerBoundaryTemperature shouldBe Temperature(18, Celsius)
-      thermalHouse.upperBoundaryTemperature shouldBe Temperature(22, Celsius)
+      )).toKilowattHours shouldBe 2741.5
```

Comment:
Rather pedantic, I know, but I would prefer to have the unit here tested as well => Quantity.getQuanties(1d, PowerSystemUnits.WATTHOUR). Then you can use should approximate as well.
Same for the others.
If you do so, there will be a conflict of different types as there are `squants.Energy` (used in SIMONA) and `ComparableQuantity[javax.measure.quantity.Energy]` (used in PSDM). But if you write one of them not only as Energy but e.g. as squants.Energy, this should work.